pkg/stanza/operator/input/windows/event_test.go

```diff
@@ -7,7 +7,12 @@
 package windows
 
 import (
+	"os"
+	"path/filepath"
 	"testing"
+	"unsafe"
+
+	"golang.org/x/text/encoding/unicode"
```

Comment:
I think this will fail lint because it should be in one group with the testify package.